lib-multisrc/mangathemesia/src/eu/kanade/tachiyomi/multisrc/mangathemesia/MangaThemesia.kt

```diff
@@ -338,18 +404,25 @@ abstract class MangaThemesia(
             return
         }
 
-        val request = countViewsRequest(document) ?: return
-        runCatching { client.newCall(request).execute().close() }
+        try {
+            val request = countViewsRequest(document) ?: return
+            client.newCall(request).enqueue(countViewsCallback)
+        } catch (_: Exception) { }
+    }
+
+    private val countViewsCallback = object : Callback {
+        override fun onResponse(call: Call, response: Response) = response.close()
+        override fun onFailure(call: Call, e: IOException) = Unit
```

Comment:
Create the callback on site, it's cheap enough. And we don't need the `try-catch` block.

Reply:
Could use `val callback = object : Callback {`, that's less indentation.